src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -621,12 +644,28 @@ export default defineComponent({
       showToast(this.$t('Video.Video has been removed from your saved list'))
     },
 
+    hideChannel: function(channelName) {
+      const hiddenChannels = JSON.parse(this.$store.getters.getChannelsHidden)
+      hiddenChannels.push(channelName)
+      this.updateChannelsHidden(JSON.stringify(hiddenChannels))
+
+      showToast(this.$t('Channel Hidden', { channel: channelName }))
+    },
```

Comment:
Please change this pull request to use the channel ID, not the name. Blocking by name is unreliable as the channel owner can change the channel name, so you would have to block it again when that happens, but the ID never changes, so it will continue to work even when the name changes.

Reply:
I went with the name since the Ids were very unreadable, if #4230 merges though that's no longer a concern, I can switch it Ids